CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+# Contributing guide
+
+**First off all, thank you for taking the time to contribute into Prospero!** The below contents will help you through the steps for getting started with Prospero. Please make sure to read the relevant section before making your contribution. It will make it a lot easier for us maintainers and smooth out the experience for all involved. The community looks forward to your contributions.
+
+## Documentation
+
+You can find all information and examples in [Prospero documentation](https://docs.wildfly.org/prospero/)
+
+## Development environment
+
+* Maven 3.9.x build tool https://maven.apache.org/, other versions may work too
+* JDK 17+ You can use any distribution
```

Comment:
I believe the minimum JDK version here is 11, based on this config: https://github.com/wildfly-extras/prospero/blob/1.1.x/pom.xml#L489

The 1.3.x and the main branches look to be JDK 17.